### DISCUSSION TITLE
[Feature] Sort "Shared albums" section by most recent photo

### DISCUSSION THREAD

**Discussion by usikpavel** [immich-app/immich#3914] [Feature Request]
### The feature

I shared a lot of albums with my mom.
**Feature:** mom should see the albums ordered by most recent photo date in "Sharing" section. Also, it should be proper for me to see the shared albums in ordered form.

There is no any ordering in shared albums now in v1.74.0.

### Platform

- [X] Server
- [x] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1689

**Reply by usikpavel** [reply]
It's not a duplicate. Check "Sharing" section

**Reply by usikpavel** [reply]
Screenshot, unordered "Sharing" albums:

![image](https://github.com/immich-app/immich/assets/18679813/cc518ab1-e533-4004-9ff5-66c8466a99e4)

Screenshot, ordered "Albums" section:

![image](https://github.com/immich-app/immich/assets/18679813/3348894a-4c98-4101-8991-400604167127)

**Reply by usikpavel** [reply]
There is no any option to sort shared albums by anything.

**Reply by bo0tzz** [reply]
My bad, I got this mixed up with another request last night

**Comment by bo0tzz** [comment]
These days they're ordered by recency as requested
